lldb/include/lldb/Host/ProcessLaunchInfo.h

```diff
@@ -136,7 +148,11 @@ class ProcessLaunchInfo : public ProcessInfo {
   FileSpec m_shell;
   Flags m_flags; // Bitwise OR of bits from lldb::LaunchFlags
   std::vector<FileAction> m_file_actions; // File actions for any other files
+#ifdef _WIN32
+  std::shared_ptr<PseudoConsole> m_pty;
+#else
   std::shared_ptr<PseudoTerminal> m_pty;
+#endif
```

Comment:
This can then be done uniformly as well.

Reply:
Fixed, thanks